qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
If you look at sitebase, there are now 2 redis keys - maintenance and message. Maintenance mode disables login, while message jost holds the system message. Therefore you check for maintenance mode now instead of message.